packages/storybook/src/stories/ino-chip/ino-chip.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { expect, Locator, test } from '@playwright/test';
+import { goToStory } from '../test-utils';
+
+test.describe('ino-chip', () => {
+  let chip: Locator;
+
+  test.beforeEach(async ({ page }) => {
+    await goToStory(page, ['buttons', 'ino-chip', 'default']);
+    chip = page.locator('ino-chip');
+  });
+
+  test('should render chip', async () => {
+    await expect(chip).toBeInViewport();
```

Comment:
```suggestion
    await expect(chip).toBeVisible();
```
Would probably be a better fit